xwiki/content.md

```diff
@@ -57,7 +55,7 @@ services:
       - MYSQL_USER=xwiki
       - MYSQL_PASSWORD=xwiki
   db:
-    image: "mysql:5"
+    image: "mysql:5.7"
```

Comment:
So `latest` needs to have all dependencies with a fixed versoin and it's only the main image software that doesn't have its version fixed?